index.html

```diff
@@ -11,42 +11,173 @@
+                                <span class="see-project"><a href="">See project</a></span>
+                            </div>
+                        </div>
+                        <div class="card-7 card"><h2 class="Title-post">Profesional Art Printing Data</h2>
+                            <p class="Supporting-text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                            <div class="tags">
+                                <div class="tag-link">
+                                    <span class="tag "><a href="https://www.html.org">html</a></span>
+                                    <span class="tag "><a href="https://getbootstrap.com/">bootstrap</a></span>
+                                    <span class="tag "><a href="https://www.ruby-lang.org/en/">Ruby</a></span>
+                                </div>
+                                <span class="see-project"><a href="">See project</a></span>
+                            </div>
+                            
+                        </div>
+                    </div> 
+                </section>
+                <section>
+                    <div class="About">
+                        <h2 class="about-me">About me</h2>
+                        <span><p class="description">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p></span>
+                        <span class="get-resume"><a href="">Get My Resume</a></span>
+                        <div class="image-2">
+
+                        </div>
+                            <hr class="rule-2">
+                            <div class="offer">
+                                <div class="heading">
+                                    <h3 class="head">🔸Language</h3>
+                                    <ul class="list">
+                                        <li><a href="">Javascript</a></li>
+                                        <li><a href="">Ruby</a></li>
+                                        <li><a href="">Html</a></li>
+                                        <li><a href="">Css</a></li>
+                                    </ul>
+                                </div>
+                                <div class="heading ">
+                                    <h3 class="head">🔸Frameworks</h3>
+                                    <ul class="list">
+                                        <li><a href="">Bootstrap</a></li>
+                                        <li><a href="">Ruby oR</a></li>
+                                        <li><a href="">RSPec</a></li>
+                                        <li><a href="">CapyBara</a></li>
+                                        <li><a href="">Codekit</a></li>
+                                    </ul>
```

Comment:
- [ ] Please I think it is better if you adjust the styling of this section of your project to match the Figma design and also to remove the horizontal overflow. 
### YOUR DESIGN
![Screenshot from 2023-01-14 20-56-09](https://user-images.githubusercontent.com/92303891/212496475-78b31744-cac3-4943-9a7c-69b3e1408242.png)
### FIGMA DESIGN
![Screenshot from 2023-01-14 21-02-41](https://user-images.githubusercontent.com/92303891/212496713-d3e9fbd0-6427-4ac1-89aa-0235007a2804.png)